sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlanInfo.scala

```diff
@@ -71,6 +77,8 @@ private[execution] object SparkPlanInfo {
       plan.nodeName,
```

Comment:
We can construct the name with `codegenStageId` here.

Reply:
then the diff can be much smaller.

Reply:
Thanks for reviwing. The proposal sounds good to me.